src/components/Sidebar/Sidebar.tsx

```diff
@@ -1,4 +1,4 @@
-import { Box } from '@mui/material'
+import {Box} from '@mui/material'
```

Comment:
This isn't really important but in js it's best practice to do { thing } with the spaces, it'll get fixed with the lint mr